src/calculator.ts

```diff
@@ -0,0 +1,69 @@
+import fs from "fs";
+import { linearQF, Contribution, Calculation } from "pluralistic";
+
+type AugmentedResult = Calculation & {
+  projectName: string;
+  payoutAddress: string;
+};
+
+export default class Calculator {
+  private baseDataPath: string;
+  private chainID: string;
```

Comment:
I am happy with either but I just want to make sure make a decision now and keep it consistent

Reply:
@boudra I really like the uppercase one, but I asked chat GPT 4 and he told me exactly what you said, so it's 2 against 1, I will change it :(

its reply:
> In a TypeScript project, it's recommended to follow the standard JavaScript naming conventions, which include using camelCase for variable names. Therefore, you should use projectId instead of projectID. This approach maintains consistency and readability across your codebase.

Reply:
:)